website/static/js/fangorn.js

```diff
@@ -2090,13 +2100,15 @@ tbOptions = {
     onmouseoverrow : _fangornMouseOverRow,
     sortDepth : 2,
     dropzone : {                                           // All dropzone options.
+        maxFilesize: 10000000,
```

Comment:
Dropzone tries to internally handle the file size cap but we handle it externally.
So I just set it really high so that it won't be an issue any more.